docs/deployment/helm.md

```diff
@@ -65,6 +65,61 @@ export NFD_NS=node-feature-discovery
 helm install nfd/node-feature-discovery --set nameOverride=NFDinstance --set master.replicaCount=2 --namespace $NFD_NS --create-namespace
 ```
 
+## Upgrading the chart
+
+To upgrade the `node-feature-discovery` deployment to a new version via Helm.
+
+### From v0.7 and older
+
+Please see the [Uninstall guide](https://kubernetes-sigs.github.io/node-feature-discovery/v0.7/get-started/deployment-and-usage.html#uninstallation).
+And then install using the documented [Installation guide](https://kubernetes-sigs.github.io/node-feature-discovery/master/deployment/helm.html).
+
+### From v0.8 - v0.11
+
+Helm deployment of NFD was introduced in v0.8.0.
+
+```bash
+export NFD_NS=node-feature-discovery
+# Uninstall the old NFD deployment
+helm uninstall node-feature-discovery --namespace $NFD_NS
+# Update Helm repository
+helm repo update
+# Install the new NFD deployment
+helm upgrade --install node-feature-discovery nfd/node-feature-discovery --namespace $NFD_NS --create-namespace --set master.enable=false
+helm upgrade --install node-feature-discovery nfd/node-feature-discovery --namespace $NFD_NS --create-namespace --set master.enable=true
+```
+
+### From v0.11 - v0.13
+
+During version v0.12 the CRD `NodeFeature` was introduced, but not enabled by default.
+
+```bash
+export NFD_NS=node-feature-discovery
+# Update Helm repository
+helm repo update
+# Install CRD's
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/node-feature-discovery/master/deployment/base/nfd-crds/nfd-api-crds.yaml
+# Install the new NFD deployment
+helm upgrade node-feature-discovery nfd/node-feature-discovery --namespace $NFD_NS --set master.enable=false
```

Comment:
I'd suggest to split this here. There needs to be a "pause", wait for all nfd-workers to upgrade and be in READY state